client/src/pages/Messages/Chat/useStyles.ts

```diff
@@ -0,0 +1,133 @@
+import { Theme } from '@mui/material/styles';
+import { makeStyles } from '@mui/styles';
+const useStyles = makeStyles((theme: Theme) => ({
+  root: {
+    display: 'flex',
+    flexGrow: 8,
+    flexDirection: 'column',
+  },
+  chatContainer: {
+    marginLeft: 41,
+    marginRight: 41,
+    display: 'flex',
+    flexDirection: 'column',
+    flexGrow: 1,
+    justifyContent: 'space-between',
+  },
+  root1: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+    height: 89,
+    marginBottom: 34,
+    boxShadow: '0 2px 20px 0 rgba(88,133,196,0.10)',
+  },
+  root2: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'flex-end',
+  },
+  root3: {
+    display: 'flex',
+  },
+  content: {
+    display: 'flex',
+    alignItems: 'center',
+    marginLeft: 24,
+  },
+  username: {
+    fontSize: 20,
```

Comment:
Let's resort to using variants to ensure a consistent font size across the app :) 

Ditto where applicable